subsys/llext/llext_link.c

```diff
@@ -64,6 +112,7 @@ static int llext_get_symbol(struct llext *ext, void *arg)
 
 	if (addr) {
 		se->addr = addr;
+		se->ext = ext;
```

Comment:
a bit confusing, why is the extension assigned to the struct when its passed in as a parameter already